ui/src/router/index.ts

```diff
@@ -29,9 +32,26 @@ const router = createRouter({
       component: DataTableDemo
     },
     {
-      path: '/dataTableDemo/edit/:id',
-      name: 'Edit',
-      component: EditNode
+      path: '/provisionConfig',
+      name: 'provisionD',
+      component: provisionD,
+      children: [
+        {
+          path: '/provisionConfig/reqDefinition',
+          name: 'req',
+          component: reqDefForm
+        },
+        {
+          path: '/provisionConfig/threadPools',
+          name: 'thread',
+          component: threadPoolsForm
+        },
+        {
+          path: '/provisionConfig/reqDefinition/edit/:id',
+          name: 'Edit',
+          component: EditNode
+        }
```

Comment:
This works, but child paths don't need the parent in the string. Could change these to just `reqDefinition`, `threadPools`, and `reqDefinition/edit/:id'`

Reply:
Removed parent string from child